src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/builder/policy/PolicyViolationMessageBuilder.java

```diff
@@ -88,7 +88,7 @@ public String getNotificationType() {
     }
 
     @Override
-    public List<ProviderMessageContent> buildMessageContents(Long notificationId, Date providerCreationDate, ConfigurationJobModel job, RuleViolationNotificationView notificationView,
+    public List<ProviderMessageContent> buildMessageContents(Long notificationId, Long providerConfigId, Date providerCreationDate, ConfigurationJobModel job, RuleViolationNotificationView notificationView,
```

Comment:
Since we are passing in the providerConfigId to all of these Builders, would it make more sense to pass in a ProviderMessageContent.Builder() instead with all of the common parts already filled in. Then use that as the base of all the individual ProviderMessageContent.Builder's we create?

Reply:
This should still be the method that calls the `build()` method if we do go that route.